src/dstr-assignment/obj-rest-non-string-computed-property.case

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2020 HyeockJin Kim. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: >
+    Destructuring field can be a non-string computed property, i.e it
+    can be defined only at runtime. Rest operantion needs to skip
+    these properties as well.
+template: default
+esid: pending
+includes: [propertyHelper.js]
+features: [object-rest]
+---*/
+
+//- setup
+var a = 1.;
+var b, rest;
+//- elems
+{[1.]:b, ...rest}
```

Comment:
I would prefer `{[a]: b` here, as it is a bit more readable.
It would be nice to have more combinations in future:

| `elems` | `vals` |
|-------|------|
| `1.0`   | `1`    |
| `1.`    | `[1]`  |
| `"1"`   | `1.0`  |
| `[1]`   | `"1"`  |
| `1e0`   | `[1]`  |
| `[1e0]`   | `1.0`  |

Reply:
Then, should I create and upload another `.case` file for the combination you presented?